loops.js

```diff
@@ -45,7 +51,15 @@
 // >> 9
 // ==========================================
 
-
+for (let i = 0;i<10;i++){
+    if (i<3){
+        console.log(i);
+    } if (i>2 && i<6) {
+        console.log("> "+i);
+    } if (i>5){
+        console.log(">> "+i)
+    }
+}
 
 
```

Comment:
je hebt bij deze opdracht alleen if- statements gebruikt voor nu is dit goed gegaan, wanneer je een gecompliceerde code gaat schrijven denk ik dat het handig is dat je in je code ook else gebruikt.